troposphere/settings/__init__.py

```diff
@@ -1,4 +1,5 @@
+from troposphere.settings.default import *
 try:
-    import troposphere.settings.local
+    from troposphere.settings.local import *
```

Comment:
That is the desired effect. I just change it back to what it looked like before, and it works fine.

https://github.com/CCI-MOC/GUI-Frontend/pull/9/files
And do a search for "__init__"